pkg/windows/windows.go

```diff
@@ -205,27 +204,7 @@ func New(workerIgnitionEndpoint, vxlanPort string, instanceInfo *instance.Info,
 // Interface methods
 
 func (vm *windows) GetIPv4Address() (string, error) {
-	if ip := net.ParseIP(vm.instance.Address); ip != nil {
```

Comment:
Please call out that you are doing this refactoring in the commit message and why you are doing so.